src/mango/src/mango_idx_view.erl

```diff
@@ -124,7 +124,13 @@ is_usable(Idx, Selector, SortFields) ->
     % we don't need to check the selector for these
     RequiredFields1 = ordsets:subtract(lists:usort(RequiredFields), lists:usort(SortFields)),
 
-    mango_selector:has_required_fields(Selector, RequiredFields1)
+    % _id and _rev are implicitly in every document so
+    % we don't need to check the selector for these either
+    RequiredFields2 = ordsets:subtract(
+        lists:usort(RequiredFields1), 
```

Comment:
`RequiredFields1` just came out of `ordsets:subtract/2` up a line, it's an ordset itself, so you don't need `usort` it.